utils/release_helper.py

```diff
@@ -0,0 +1,786 @@
+    config = configparser.ConfigParser()
+    try:
+        config.read(config_file)
+        return config[section][parameter]
+    except Exception as e:
+        print(f'{e} entry was not found - Information cannot be retrieved!')
+        exit(1)
+
+
+def create_ini_file_template(creds_file):
+    try:
+        new_creds_file = os.path.expanduser(creds_file)
+        with open(new_creds_file, mode='w', encoding='utf-8') as file:
+            file.write(
+                '[DEFAULT]\n'
+                'github_token = <generate your token in https://github.com/settings/tokens>\n')
+        print(f'Great! {new_creds_file} was created! Edit it to include your personal token.')
+    except Exception as e:
+        print(f'Error: {e}')
+        exit(1)
+
+
+def get_github_token(creds_file) -> str:
+    if os.path.exists(creds_file):
+        return get_parameter_from_ini(creds_file, "DEFAULT", "github_token")
+    else:
+        print(f'{creds_file} file was not found!')
+        if get_confirmation(f'Would you like to create the "{creds_file}" file?'):
+            create_ini_file_template(creds_file)
+        exit(1)
+
+
+def create_github_session(args) -> object:
+    return Github(get_github_token(args.creds_file))
+
+
+def get_confirmation(question) -> bool:
+    answer = str(input(f'{question} (Y/N): ')).lower().strip()
+    if answer[:1] == 'y':
+        return True
+    else:
+        return False
+
+
+def get_repo_object(session, repo_id) -> object:
+    return session.get_repo(repo_id)
+
+
+# Repository Branches
+def filter_repo_branch_by_name(branches, name) -> object:
+    return [branch for branch in branches if branch.name == name]
+
+
+def get_repo_branch(repo, branch_name) -> object:
+    try:
+        return repo.get_branch(branch_name)
+    except Exception as e:
+        print(f'Error: {e}')
+        exit(1)
+
+
+def get_repo_branches(repo) -> list:
+    return repo.get_branches()
+
+
+def get_repo_stable_branch(repo) -> object:
+    branches = get_repo_branches(repo)
+    return filter_repo_branch_by_name(branches, "stable")
+
+
+def get_old_stabilization_branch(repo) -> object:
+    branches = get_repo_branches(repo)
+    latest_version = get_latest_version(repo)
+    branch_name = f'stabilization-v{latest_version}'
+    return filter_repo_branch_by_name(branches, branch_name)
+
+
+def remove_old_stabilization_branch(repo, branch) -> None:
+    if get_confirmation(f'Are you sure about removing the "{branch.name}" branch?'):
+        # https://github.com/PyGithub/PyGithub/issues/1570
+        try:
+            branch_ref = repo.get_git_ref(f'heads/{branch.name}')
+            branch_ref.delete()
+        except Exception as e:
+            print(f'Error: {e}')
+            exit(1)
+    else:
+        print(f'Aborted! {branch.name} branch not removed.')
+
+
+# Repository Contributors
+def get_contributors_commit_diff(commit) -> str:
+    commit_diff = subprocess.run(
```

Comment:
Do you think it would make sense to ensure that we are in the root directory of the project? In theory, I can run the script from any place on the file system and I am afraid this could break it.

Reply:
Yes, as the previous comment about the location, it should be more robust. I will take a look on this.

Reply:
It was fixed by using the `cwd` parameter in `subpress.run`.